content/docs/apps/custom-domains.md

```diff
@@ -69,47 +69,4 @@ Or:
 
 ## Managed Service Method
 
-See the [CDN Route service page]({{< relref "docs/services/cdn-route.md" >}}) for more information.
-
-cloud.gov offers a managed service that allows you to forward a domain you control to a cloud.gov application using [AWS CloudFront](https://aws.amazon.com/cloudfront/). Traffic is encrypted using an SSL certificate generated by [Let's Encrypt](https://letsencrypt.org/).
-
-1. Target the space your application is running in.
-
-    ```bash
-    $ cf target -o <org> -s <space>
-    ```
-
-1. Create a service instance. Replace *my-app.app.cloud.gov* with the  domain that has been automatically assigned to your app by cloud.gov.
-
-    ```
-    $ cf create-service cdn-route cdn-route my-cdn-route \
-        -c '{"domain": "my.domain.gov", "origin": "my-app.app.cloud.gov"}'
-
-    Create in progress. Use 'cf services' or 'cf service my-cdn-route' to check operation status.
-    ```
-
-    If you have more than one domain you can pass a comma-delimited list to the `domain` parameter, just keep in mind that the broker will wait until all domains are CNAME'd:
-
-    ```
-    $ cf create-service cdn-route cdn-route my-cdn-route \
-        -c '{"domain": "my.domain.gov,www.my.domain.gov", "origin": "my-app.app.cloud.gov"}'
-
-    Create in progress. Use 'cf services' or 'cf service my-cdn-route' to check operation status.
-    ```
-
-1. Get the DNS instructions.
-
-    ```
-    $ cf service my-cdn-route
-
-    Last Operation
-    Status: create in progress
-    Message: Provisioning in progress; CNAME domain "my.domain.gov" to "d3kajwa62y9xrp.cloudfront.net."
-    ```
-
-1. Repeat last step until the CloudFront distribution is provisioned. Note this can take up to 30 minutes.
-1. Create/update your DNS configuration.
-    * In AWS Route 53, this will (confusingly) be an `A` record with an `Alias` to `<something>.cloudfront.net`.
-    * If you can, wait to **do this step during low-traffic hours**. It can take up to 15 minutes before the certificate is in place, and visitors will get a certificate error when visiting the custom domain in the meantime.
-1. Wait for the DNS changes to propagate.
```

Comment:
I don't think we need route53 here--it's potentially confusing, since not all users are going to use it at all. The low-traffic point is also pretty opinionated--maybe it's useful to say that opening the domain will show an invalid certificate until the changes propagate through aws.